libecho/src/BitAudioConverter.h

```diff
@@ -29,7 +29,7 @@ class BitAudioConverter : public IAudioConverter<T> {
         for (int i = 0; i < 8; i++) {
             int bit = (data >> i) & 1;
             int freq = (bit == 1) ? hiFreq : loFreq;
-            for (int t = 0; t < this->outputFormat.sampleRate() * this->windowLength; t++) {
+            for (int t = 0; t < this->windowSize; t++) {
```

Comment:
I think that is code written by @Riyria-was-taken , are you sure that is the right change?

Reply:
yes, see below